cypress/support/systemIntake.js

```diff
@@ -0,0 +1,56 @@
+cy.systemIntake = {
+  contactDetails: {
+    fillNonBranchingFields: () => {
```

Comment:
This function fills out every single field that does not have an effect on the rest of the form. The fields filled out here are independent. For example, the requester's name doesn't have an impact on any other field. The ISSO field is **not** included here because if the intake request has an ISSO, they need to fill their name.